course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -83,6 +84,7 @@
                         roleName: $scope.getRoleName(user.role_id),
                     });
                 });
+            $scope.searchInProgress = false;
```

Comment:
i was looking for a finally or always but angular doens't seem to have any. chaining then will do it though :)